packages/fhir-converter/src/lib/cda/cda.js

```diff
@@ -130,7 +129,8 @@ module.exports = class cda extends dataHandler {
     }
     res = res.replace(elementTime00010101Regex, elementTime00010101Replacement);
     res = res.replace(valueTime00010101Regex, valueTime00010101Replacement);
-    res = res.replace(singleAmpersandRegexOneOff, singleAmpersandReplacementOneOff);
+    res = res.replace(ampersandRegex, "&amp;");
```

Comment:
How do we know this is the right replacement? Or are we just assuming the intended suffix was `amp;`?